software/controller/lib/non_volatile/nvparams.h

```diff
@@ -50,7 +50,7 @@ struct Structure {
 
   uint8_t reserved{0};  // Kept here for 32bits-alignment's sake
 
-  uint32_t vent_serial_number{0};
+  char vent_serial[20]{0};
```

Comment:
Maybe find a way to keep this synced with the 20 in `Debug::Variable::String<20>`